releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -4,6 +4,99 @@
 # Microsoft.FeatureManagement.AspNetCore
 [Source code ][source_code_web] | [Package (NuGet)][package_web] | [Samples][samples_web] | [Product documentation][docs]
 
+## 4.0.0-preview - January 4, 2024
+
+### Variants
+
+Variants provide the ability to surface different variations of a feature, allowing different segments of an audience to get different variations of a feature, all within a single feature flag. Variants also contain a value that can either reference another ConfigurationSection or be declared inline on the flag.
+
+```json
+{ 
+    "Name": "Big", 
+    "ConfigurationReference": "ShoppingCart:Big" 
+},  
+{ 
+    "Name": "Small", 
+    "ConfigurationValue": {
+        "Size": 300
+    }
+}
+```
+
+Variants are then assigned via Allocation, which is defined as a part of the feature flag. The fields are User, Group, Percentile, and a couple defaults.
```

Comment:
It's a delicate balance between to create a sneak peek. I think that going over the allocation schema might be a bit much and should be left to the next step of a readme visit.

Reply:
Reworked it around your other comments as well. Still included a small snippet about allocation, let me know what you think.